backend/api/utils.py

```diff
@@ -1,31 +1,11 @@
-"""Retrieves and returns applications keys."""
+"""Api's utils module"""
 
-import json
+import os
 
+import dotenv
 
-def get_api_key(key: str) -> str:
-    """
-    Retrieves and returns an API key
 
-    Args:
-        key (str): Key name for API key saved in file.
-
-    Returns:
-        str: API key
-    """
-    with open(r"api\secrets\api_keys.json") as api_keys:
-        return json.loads(api_keys.read()).get(key)
-
-
-def get_secret_key(key: str) -> str:
-    """
-    Retrieves and returns secret key
-
-    Args:
-        key (str): Key name for secret key saved in file.
-
-    Returns:
-        str: Secret key
-    """
-    with open(r"api\secrets\secret.json") as secret:
-        return json.loads(secret.read()).get(key)
+def get_key(key: str) -> str:
+    """Retrieves and returns keys from .env file."""
+    dotenv.load_dotenv()
+    return os.getenv(key)
```

Comment:
`get_key` doesn't bring too much to the table. Maybe we could use `os.getenv(key)` instead of calling `get_key`?

`load_dotenv()` will load all vars from `.env` to the environment, so there is no need to do it every time we want some env. It should be enough to run it in `settings.py` once.

For the local development we would like to have a fresh env for every restart, so we could add flag `override` 
`load_dotenv(override=True)`